### DISCUSSION TITLE
难评

### DISCUSSION THREAD

**Discussion by Yizakl** [share121/inter-knot#5255] [其他]
![IMG_0122](https://github.com/user-attachments/assets/dc093e39-2de6-45e3-acaf-578cb97a9ec6)
有必要举办吗qwq
（部分语句因我的挠餐已删除）

**Comment by KELERSAMA** [comment]
大佬
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Reply by Yizakl** [reply]
我是fw喵

**Reply by KELERSAMA** [reply]
你是大佬喵嗷

**Comment by HoshinoHanashi** [comment]
不是直接开门就进的吗（滑稽）

**Reply by Yizakl** [reply]
确实，但是Github毕竟是国外网站
有时候不能直接进

**Reply by obai-gs** [reply]
emm那本来就在外面的怎么算

**Comment by fzdqz** [comment]
裸连嘛(￣￢￣*）沿海切用户接口确实可以直接连

**Reply by Yizakl** [reply]
雀氏，我人在胡九雀氏大部分时候是裸连（
但是这个真的有必要举报吗qwq

**Reply by fzdqz** [reply]
没必要啊∠( ᐛ 」∠)_

> 雀氏，我人在胡九雀氏大部分时候是裸连（ 但是这个真的有必要举报吗qwq
